website/docs/core-concepts/connecting-to-data-sources/README.md

```diff
@@ -1,5 +1,41 @@
 # Connecting to Datasource
 
+## **Outline**
+
+**Introduction** 
+
+**Prerequisites​**
+
+Before you begin, you'll need to create an account on Appsmith Cloud, or you can self-host Appsmith locally or on a hosting platform of your choice.
+
+**How to connect to a datasource**
+
+Explain how to connect to a datasource
+
+**APIs**
+* Information about APIs
+* List of supported APIs. 
+* Sample API​ (Explain how to use sample API)
+
+**Databases**
+* Information about databases
+* List of supported Database.
+* Sample DB​ (Explain how to use sample DB, both movies and users)
+
+**Security**
+
+**Internal database​(LocalDB)**
```

Comment:
I think this should come above Security and lets rename it as 'Connect to local datasource'

Reply:
updated

Reply:
@danciaclara - Why we want to specify it to local datasource. User's could use a hosted service/free services too. Local word would be misleading here. I think we should keep is `create a datasource connection`.

Reply:
Wait this section is about local db/api right? Thats why I suggested changing the heading. 
https://appsmith-docs-git-connecting-to-datasource630-get-appsmith.vercel.app/core-concepts/connecting-to-data-sources/#connect-to-a-localhost-database-api

On another note, I feel this is redundant info from this page -  https://docs.appsmith.com/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith Can't we have this linked there